apps/frontend/src/app/5_pages/StakePage/components/VestingStakesFrame/components/AdjustVestingStakeRenderer.tsx

```diff
@@ -24,6 +26,15 @@ export const AdjustVestingStakeRenderer: FC<Vesting> = ({
     false,
   );
 
+  const { endDate } = useGetVestingStakeStartEndDates(vestingContract);
+
+  const currentDate = useMemo(() => new Date(), []);
+
+  const isDisabled = useMemo(
+    () => currentDate.getTime() / MS > Number(endDate),
+    [endDate, currentDate],
+  );
```

Comment:
```suggestion
  const currentDate = useMemo(() => dayjs().unix(), []);

  const isDisabled = useMemo(
    () => currentDate > Number(endDate),
    [endDate, currentDate],
  );
```
You can just use `unix` function from `dayjs` to get a timestamp.